sql-update/make-yoder.sql

```diff
@@ -0,0 +1,7 @@
+update "actors"
+set "firstName" = 'baby',
+    "lastName" = 'yoda'
+where "actorId" = 15;
+
+select *
+FROM "actors";
```

Comment:
Instead of doing a separate query, it is more useful to use the `returning` keyword to return the updated row(s)